lnwire/message_test.go

```diff
@@ -453,7 +454,24 @@ func newMsgFundingLocked(t testing.TB, r io.Reader) *lnwire.FundingLocked {
 	pubKey := randPubKey(t)
 
 	msg := lnwire.NewFundingLocked(lnwire.ChannelID(c), pubKey)
-	msg.ExtraData = createExtraData(t, r)
+
+	// When testing the WriteMessage function we need to populate
+	// the alias to test the encoding of the tlv_stream in the
+	// funding_locked message (now called channel_ready in the bolt spec).
+	_, err = r.Read(c[:8])
+	require.NoError(t, err, "unable to generate channel alias")
+	aliasScid := lnwire.NewShortChanIDFromInt(
+		binary.BigEndian.Uint64(c[:8]),
+	)
+
+	msg.AliasScid = &aliasScid
```

Comment:
I think we could simplify this code a bit.  Something like:

```go
msg := lnwire.FundingLocked{
  ChanID: lnwire.ChannelID(c),
  NextPerCommitmentPoint: pubkey,
  AliasScid: lnwire.NewShortChanIDFromInt(r.Int63()),
  ExtraData: make([]byte, 0),
}
```

The `RecordProducer` stuff below to populate `ExtraData` could stay the same.

Reply:
Done, Though I had to keep the aliasScid separate because the `AliasScid` is a pointer but the `NewShortChanIDFromInt` returns a value type